### PR TITLE
vscode: Don't use CMakePresets.json workflow

### PR DESCRIPTION
**Pull request by juan-lunarg**
Our repo doesn't work well with the CMakePresets.json workflow vscode encourages, and can be actively confusing/hindering to new contributors.

We actually discussed this a while ago internally.

Provide this default settings.json so vscode uses the default CMake workflow.

### CODE REVIEW

**Comment by juan-lunarg** [comment]
> How will this affect repo's that already _have_ .vscode/settings.json's?

It will overwrite the existing settings.json currently as is.

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "cmake.useCMakePresets": "never"
+}
```
Comment (juan-lunarg) [inline .vscode/settings.json:3]: My goal is to share this setting among users. Not sure how to do this elegantly / idiomatically.

**Comment by charles-lunarg** [comment]
> It will overwrite the existing settings.json currently as is.

Thats quite annoying, since I have a lot of settings set per-repo.

**Comment by juan-lunarg** [comment]
> Thats quite annoying, since I have a lot of settings set per-repo.

Agreed. Closing for now. Might be better for everyone to just remove CMakePresets.json from this repo.

**Comment by juan-lunarg** [comment]
https://github.com/KhronosGroup/Vulkan-ValidationLayers/pull/6396

**Pull request closed without merging**